resdk/tests/functional/duplication/e2e_duplication.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring
+from resdk.exceptions import ResolweServerError
+from resdk.tests.functional.base import BaseResdkFunctionalTest
+
+
+class TestDuplication(BaseResdkFunctionalTest):
+
+    def test_collection_duplication(self):
+        collection = self.res.collection.create(name='Test collection')
+        duplicate = collection.duplicate()
+        self.assertEqual(duplicate.name, 'Copy of Test collection')
+
+    def test_sample_duplication(self):
+        sample = self.res.sample.create(name='Test sample')
+        duplicate = sample.duplicate()
+        self.assertEqual(duplicate.name, 'Copy of Test sample')
+
+    def test_sample_duplication_perms(self):
```

Comment:
IMO, this test does not belong here. In resdk, we should only verify that Resolwe endpoints work with a minimal example (e.g. creating a duplicated data/sample/collection). All edge cases, permissions, etc... should be tested in Resolwe.

Reply:
This test is there to check what happens if permission error is raised. @dblenkus suggested adding a `PermissionError(ResolweServerError)` to `exceptions.py ` and raising that if resolwe returns permission error.

Reply:
OK, this sounds fine.